newIDE/app/src/EventsSheet/index.js

```diff
@@ -2140,6 +2152,17 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
                     }
                   />
                 )}
+                {this.state.layoutVariablesDialogOpen && (
+                  <GlobalAndSceneVariablesDialog
+                    projectScopedContainersAccessor={
+                      projectScopedContainersAccessor
+                    }
+                    open
+                    onCancel={() => this.editLayoutVariables(false)}
+                    onApply={() => this.editLayoutVariables(false)}
+                  />
+                )}
+                )}
```

Comment:
is formatting right?

Reply:
But you've opened only 1 `{(` I don't understand why there are 2 `)}` here, 1 line should not be here? This is very strange

Reply:
Indeed, it work without it. It's strange that there wasn't any syntax error.

Reply:
I think this is because it's understood as characters, like:

`<div>)}</div>`

which is valid (but will display them somewhere in the interface, which is obviously a problem!)